### PR TITLE
Check reminders/feedback every hour

### PR DESCRIPTION
**Pull request by henniska**
AWS lambda not changed (the rate is still 7 days).

### CODE REVIEW

File: src/util/sessions-dynamodb-provider.js [hunk 1]
```diff
@@ -25,7 +25,18 @@ module.exports = class DynamoDBProvider {
   }
 
   readAllWithReminders() {
-    let currentDay = strings['@WEEKDAYS'][new Date().getDay()].toUpperCase();
+    let d = new Date();
+    let utc = d.getTime() + (d.getTimezoneOffset() * 60 * 1000);
+    // UTC + 2 -> Suomen aika
+    let currentDate = new Date(utc + (2 * 60 * 60 * 1000));
+    let currentDay = strings['@WEEKDAYS'][currentDate.getDay()].toUpperCase();
+    let currentHourWithoutZero = currentDate.getHours();
```
Comment (veituomi) [inline src/util/sessions-dynamodb-provider.js]: let currentHourWithZero = ('0' + currentDate.getHours()).substr(-2, 2);

File: src/util/sessions-dynamodb-provider.js [hunk 2]
```diff
@@ -36,14 +47,39 @@ module.exports = class DynamoDBProvider {
     };
 
     return this.table.scan(params).then((items) => {
-      log.debug('Sessions with reminder: ' + JSON.stringify(items));
-      return items;
+      let sessions = [];
```
Comment (veituomi) [inline src/util/sessions-dynamodb-provider.js]: return items.filter((item) => ('0' + item.context.time).substr(-5, 2) == currenHourWithZero);

**Review by veituomi — APPROVED** [review]
Kaapo

**Review by madkoppa — APPROVED** [review]

**Review by veituomi — APPROVED** [review]

File: src/util/sessions-dynamodb-provider.js [hunk 3]
```diff
@@ -25,7 +25,12 @@ module.exports = class DynamoDBProvider {
   }
 
   readAllWithReminders() {
-    let currentDay = strings['@WEEKDAYS'][new Date().getDay()].toUpperCase();
+    let d = new Date();
+    let utc = d.getTime() + (d.getTimezoneOffset() * 60 * 1000);
+    // UTC + 2 -> Suomen aika
+    let currentDate = new Date(utc + (2 * 60 * 60 * 1000));
+    let currentDay = strings['@WEEKDAYS'][currentDate.getDay()].toUpperCase();
```
Comment (leppaott) [inline src/util/sessions-dynamodb-provider.js:32]: getCurrentDay() could eliminate some duplicate lines

**Review by leppaott — APPROVED** [review]

**Pull request merged**